cranelift-codegen/src/isa/x86/abi.rs

```diff
@@ -39,10 +41,20 @@ struct Args {
     fpr_used: usize,
     offset: u32,
     call_conv: CallConv,
+    shared_flags: shared_settings::Flags,
+    #[allow(dead_code)]
+    isa_flags: isa_settings::Flags,
```

Comment:
@sunfishcode I added both sets of flags as we discussed; `isa_flags` is not yet used so I suppressed the warning. As I looked at it more, I realized I didn't need to implement this setting-passing for the other ISAs because the default behavior there is correct: they should split vectors because there is no SIMD implementation yet.

Reply:
Makes sense. I do think we'll ultimately have a need for the settings here, so this looks fine for now.